test/integration/targets/cloudwatchlogs/playbooks/roles/cloudwatchlogs_log_group_metric_filter/tasks/main.yml

```
@@ -0,0 +1,151 @@
+---
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- name: test cloudwatch log group metric filter
+
+  block:
+    - name: create cloudwatch log group for integration test
+      cloudwatchlogs_log_group:
+        <<: *aws_connection_info
+        state: present
+        log_group_name: ansible/integrationtest
```

resources you create should generally start with ``{{ resource_prefix }}``

I would generally create defaults/main.yml and add:
```
---
log_group_name: '{{ resource_prefix }}/integrationtest'
filter_name: '{{ resource_prefix }}/AnsibleTest'
```

Which then means:
```suggestion
        log_group_name: '{{ log_group_name }}'
```